order-book-pipeline/src/main/java/com/google/cloud/dataflow/orderbook/OrderBookProcessingPipeline.java

```diff
@@ -81,6 +81,12 @@ public interface Options extends GcpOptions {
     boolean isIncludeLastTrade();
 
     void setIncludeLastTrade(boolean value);
+
+    @Description("Maximum number of elements to output per each bundle. Total size of the data is not not exceed 2GB.")
```

Comment:
Total size of the data is not to exceed 2GB?

Reply:
Yes, that's the hard limit of bundle output in a streaming pipeline.